common/src/commonMain/kotlin/entity/DiscordGuild.kt

```diff
@@ -377,6 +380,41 @@ public data class DiscordIntegrationAccount(
     val name: String,
 )
 
+/**
+ * A representation of a [Discord Member Verification Guild structure]()
```

Comment:
Add link to: https://docs.discord.food/resources/guild#member-verification-guild-structure